EIPS/eip-draft_grant_standard.md

```diff
@@ -38,6 +39,7 @@ Contract interface for grant management (`Grant`)
     * `MAJORITY_THRESHOLD` (X% of votes necessary to unlock funds with a `minimum_token_threshold`)
     * `VOTE_THRESHOLD` (unlocks funds after X tokens signal in favor)
     * `FUND_THRESHOLD` (unlocks funds as soon as fund threshold is reached)
+      * `overflow` (optional): null by default. if set to true, allows funding over threshold
```

Comment:
* "null" is a weird default? Let's have this default to `false` if we're aiming for a boolean value
* `allow_overflow` is a better boolean name